Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -30,7 +31,11 @@ def filter(self, record):
         return False
 
 
+# Make sure we have only one SuppressWarningFilter
 v21_logger = logging.getLogger("taxii2client.v21")
+for current_filter in list(v21_logger.filters):    # pragma: no cover
+    if 'SuppressWarningFilter' in type(current_filter).__name__:
+        v21_logger.removeFilter(current_filter)
```

Comment:
Won't that remove all the filters?

Reply:
I hope we're the only ones with the `SuppressWarningFilter ` filter class.